### PR TITLE
[OCPBUGS#12282] ShiftStack cloud provider config section bugs

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.12, 4.13

Issue: https://issues.redhat.com/browse/OCPBUGS-12282

Link to docs preview: https://59374--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-installer-kuryr.html#installation-osp-setting-cloud-provider-options_installing-openstack-installer-kuryr

Link to live, broken section: https://docs.openshift.com/container-platform/4.12/installing/installing_openstack/installing-openstack-installer-kuryr.html#installation-osp-setting-cloud-provider-options_installing-openstack-installer-kuryr

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by maxwelldb** [comment]
@MaysaMacedo @stephenfin Any thoughts on this change? I want to bring the live section in line with external CCM. I'm not 100% sure about what can be salvaged.

File: modules/installation-osp-setting-cloud-provider-options.adoc [hunk 1]
```diff
@@ -7,11 +7,11 @@
...
 :_content-type: PROCEDURE
 [id="installation-osp-setting-cloud-provider-options_{context}"]
-= Setting cloud provider options
+= Setting OpenStack Cloud Controller Manager options
 
-Optionally, you can edit the cloud provider configuration for your cluster. The cloud provider configuration controls how {product-title} interacts with {rh-openstack-first}.
+Optionally, you can edit the OpenStack Cloud Controller Manager (CCM) configuration for your cluster. This configuration controls how {product-title} interacts with {rh-openstack-first}.
 
-For a complete list of cloud provider configuration parameters, see link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
+For a complete list of configuration parameters, see the "OpenStack Cloud Controller Manager reference guide" page in the "Installing on OpenStack" documentation.
```
Comment (stephenfin) [inline modules/installation-osp-setting-cloud-provider-options.adoc:14]: Do we not have the ability to cross-link our own docs? We documented all of this recently, right?

Comment (maxwelldb) [reply modules/installation-osp-setting-cloud-provider-options.adoc:14]: @stephenfin Not within a module like this, no. 

It's a limitation that everyone hates. 👍

Comment (maxwelldb) [reply modules/installation-osp-setting-cloud-provider-options.adoc:14]: And yeah, this is just to clean up some linking and language in a couple of versions of the docs that suffered a bad rebase at some point.

File: modules/installation-osp-setting-cloud-provider-options.adoc [hunk 2]
```diff
@@ -29,7 +29,7 @@ $ openshift-install --dir <destination_directory> create manifests
 $ vi openshift/manifests/cloud-provider-config.yaml
 ----
 
-. Modify the options based on link:https://v1-18.docs.kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#cloud-conf[the `cloud.conf` specification].
+. Modify the options based according to the CCM reference guide.
```
[the remark this answers is not in the document]
Comment (maxwelldb) [reply modules/installation-osp-setting-cloud-provider-options.adoc:32]:
```suggestion
. Modify the options according to the CCM reference guide.
```

**Pull request merged**